// typesafe-actions/dist/index.d 2.ts
/**
 * @name typesafe-actions
 * @author Piotr Witek <piotrek.witek@gmail.com> (http://piotrwitek.github.io)
 * @copyright Copyright (c) 2017 Piotr Witek
 * @license MIT
 */
/** Public API */
export { action } from 'typesafe-actions/dist/action';
export { createAction } from 'typesafe-actions/dist/create-action';
export { createStandardAction } from 'typesafe-actions/dist/create-standard-action';
export { createCustomAction } from 'typesafe-actions/dist/create-custom-action';
export { createAsyncAction, AsyncActionCreator } from 'typesafe-actions/dist/create-async-action';
export { createReducer } from 'typesafe-actions/dist/create-reducer';
export { getType } from 'typesafe-actions/dist/get-type';
export { isOfType } from 'typesafe-actions/dist/is-of-type';
export { isActionOf } from 'typesafe-actions/dist/is-action-of';
export { TypeConstant, Action, ActionCreator, Reducer, EmptyAction, PayloadAction, MetaAction, PayloadMetaAction, EmptyAC, PayloadAC, PayloadMetaAC, TypeMeta, ActionType, StateType, Types, } from 'typesafe-actions/dist/type-helpers';
export { createActionDeprecated } from 'typesafe-actions/dist/create-action-deprecated';
